.circleci/config.yml

```diff
@@ -1,24 +1,23 @@
 version: 2.1
 
 orbs:
-  node: circleci/node@4.9.0
   browser-tools: circleci/browser-tools@1.2.4
 
+executors:
+  node_lts:
+    docker:
+      - image: cimg/node:18.16-browsers
+        environment:
+          NODE_OPTIONS: "--openssl-legacy-provider"
+  node16_20:
+    docker:
+      - image: cimg/node:16.20-browsers
+
 references:
 
   #
   # Workspace
   #
-  container_config_node:
-    &container_config_node
```

Comment:
**question**: was there a reason you stopped using the anchor? it's created some additional copy/pasted code

Reply:
I got ride of it initially with my lack of understanding that jobs required an executor (I was defining just in the workflow) so it looked like this anchor was only defining the `working_directory` but that isn't the case anymore. I will refactor it back in.